src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.color = color;
```

Comment:
The 'color' field is being used but it is not defined in this class. You should define it as a protected field in the Figure class, so that all subclasses can use it.